### PR TITLE
Update Dockerfile.local to support keyword pir

### PR DESCRIPTION
**Pull request by fuxingbit**
### Your checklist for this pull request
🚨Please review the [guidelines for contributing](https://github.com/primihub/community#development-workflow) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your master!
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.
- [x] Please provide a readable PR title.

### Description
Update Dockerfile.local to support keyword pir

### CODE REVIEW

File: Dockerfile.local [hunk 1]
```diff
@@ -6,9 +6,13 @@ ENV DEBIAN_FRONTEND=noninteractive
 ENV PYTHONUNBUFFERED=1
 
 RUN  apt-get update \
-  && apt-get install -y python3 python3-dev libgmp-dev python3-pip git \
+  && apt-get install -y python3 python3-dev libgmp-dev python3-pip libtool pkg-config git \
   && rm -rf /var/lib/apt/lists/*
 
+WORKDIR /opt
+RUN git clone https://gitee.com/primihub/libzmq.git \
```
Comment (LinuxSuRen) [inline Dockerfile.local:13]: Why use Gitee instead of GitHub?

**Review by LinuxSuRen — CHANGES_REQUESTED** [review]
And we should build the `Dockerfile.local` in the GitHub Actions to make sure it works well.

File: Dockerfile.local [hunk 2]
```diff
@@ -33,7 +37,7 @@ COPY ./src/primihub/protos/ ./src/primihub/protos/
 
 WORKDIR /app/python
 RUN python3 -m pip install --upgrade pip \
-  && python3 -m pip install -r requirements.txt -i https://pypi.tuna.tsinghua.edu.cn/simple \
+  && python3 -m pip install -r requirements.txt -i http://mirrors.aliyun.com/pypi/simple \
```
[the remark this answers is not in the document]
Comment (LinuxSuRen) [reply Dockerfile.local:40]: By the way, what is the reason to change the service address? `pypi.tuna.tsinghua.edu.cn` still can work well. See also:

```shell
curl -k -I https://pypi.tuna.tsinghua.edu.cn/simple/
HTTP/2 200
server: nginx/1.18.0
date: Mon, 14 Nov 2022 02:04:50 GMT
content-type: text/html
content-length: 24356098
last-modified: Mon, 14 Nov 2022 02:00:29 GMT
etag: "6371a13d-173a502"
accept-ranges: bytes
```

File: Dockerfile.local (hunk 1 above)
Comment (fuxingbit) [reply Dockerfile.local:13]: Dockerfile.local Only used by local build images

File: Dockerfile.local (hunk 2 above)
[the remark this answers is not in the document]
Comment (fuxingbit) [reply Dockerfile.local:40]: aliyun fast download speed

**Review by phoenix20162016 — APPROVED** [review]

**Review by LinuxSuRen — APPROVED** [review]

**Pull request merged**